### ISSUE TITLE
Misleading examples: the Readable Stream should be closed

### ISSUE THREAD

**Issue by pietrovismara** [leetreveil/musicmetadata#130]
The examples on the README.md file do something like this:
```
mm(fs.createReadStream('sample.mp3'), { duration: true }, function (err, metadata) {

});
```
Doing this way, the readable stream is left open, and creates a leak. After scanning hundreds of files, my app almost freezes and i have various problem. 

I don't know if this is a bug or if it's working as intended, i believe the best would be `mm` automatically closing the stream when it has finished parsing.

Anyway the correct example atm should be:
```
        var fileStream = fs.createReadStream(path);
        mm(fileStream, (err, metadata) => {
            fileStream.close();
        });
```

Am i wrong?

**Comment by leetreveil** [comment]
The library was originally designed to keep the stream open for further processing. In hindsight, this was probably the wrong decision.

Happy to take a pull request that either, A) automatically closes the stream or B) updates the docs with info on how to close the file stream.

**Comment by pietrovismara** [comment]
I think the best is to left the stream open to give the chance to make more things with stream, but i'll update the docs and make you a pull request as soon as i can.

**Comment by leetreveil** [comment]
Fixed in 7a56f484a71a4303d95ba36abf969d2b28c238b7
